### PR TITLE
Fix: Check tutor type on page revisit before displaying inline feedback (fixes #91)

### LINKED ISSUE #91: Completed question component feedback popups opening up on page load
### Subject of the issue/enhancement/features
Question component feedback popups open automatically if you leave the page that contains a completed question and navigate back to it (this includes going to the menu and then going back to the page and on an LMS or using scorm_test_harness.html).

### Your environment
The latest Framework version 5.28.15 but it also happens some slightly older versions.
It happens on Chrome, Firefox and Edge.
Confirmed on Windows and Android but I don't think it's a browser specific issue.

### Screenshots (if you can)
Here's a video
https://user-images.githubusercontent.com/110032382/233314872-994283d5-c98c-4270-b063-23c09823742f.mp4

### PR DESCRIPTION
**Pull request by joe-allen-89**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
Fixes (#91)

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
*  Check tutor type on page revisit before displaying inline feedback (#91)

### CODE REVIEW

**Review by kirsty-hames — APPROVED** [review]
Can confirm fix testing on Mac safari/FF/Chrome and iPhone.

File: js/adapt-contrib-tutor.js [hunk 1]
```diff
@@ -14,23 +14,30 @@ class Tutor extends Backbone.Controller {
...
   onButtonsViewPostRender(view) {
     const { model } = view;
-    const config = (!model.get('_tutor') || (model.get('_tutor')?._isInherited ?? true))
-      ? Adapt.course.get('_tutor')
-      : model.get('_tutor');
-    if (!config) return;
-    const type = TUTOR_TYPE(config._type?.toUpperCase());
-    if (type !== TUTOR_TYPE.INLINE) return;
+    const type = this.getTutorType(model);
+    if (!type || type !== TUTOR_TYPE.INLINE) return;
```
Comment (eleanor-heath) [inline js/adapt-contrib-tutor.js:40]:
```suggestion
    if (type !== TUTOR_TYPE.INLINE) return;
```

not sure if `!type` condition is needed

**Review by guywillis — APPROVED** [review]
Tested in Chrome, working as expected

**Review by eleanor-heath — APPROVED** [review]
approved - left a couple of minor comments

File: js/adapt-contrib-tutor.js [hunk 2]
```diff
@@ -14,23 +14,30 @@ class Tutor extends Backbone.Controller {
     });
   }
 
+  getTutorType(model) {
+    const config = (!model.get('_tutor') || (model.get('_tutor')?._isInherited ?? true))
+      ? Adapt.course.get('_tutor')
+      : model.get('_tutor');
```
[the remark this answers is not in the document]
Comment (joe-allen-89) [reply js/adapt-contrib-tutor.js:20]: Agreed, was copied across but does shorten nicely.

**Review by oliverfoster — APPROVED** [review]

**Review by joe-replin — APPROVED** [review]
Tested in FF. Working as expected.

**Pull request merged**